docs/book/src/user/quick-start.md

```diff
@@ -0,0 +1,71 @@
+# Quick Start
+
+In this tutorial we'll cover the basics of how to use Cluster API to create one or more Kubernetes clusters.
+
+## Prerequisites
+
+- Install and setup [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) in your local environment.
+- A working Kubernetes cluster to be used as [management cluster](../reference/glossary.md#management-cluster).
+
+## Before we start
+
+#### Picking Infrastructure and Bootstrap Providers
+
+The [providers section](../reference/providers.md) offers a quick reference to some providers
+available and compatible with Cluster API _v1alpha2_.
+
+<aside class="note">
+<h1>Following Along vs Jumping Ahead</h1>
+
+If you're unfamiliar with the different providers types introduced in v1alpha2
+and how they interact with each other, please refer to the [user guide concepts].
+
+[user guide concepts]: ../user/concepts.md
+</aside>
+
+To proceed, we'll need to pick and choose a _Bootstrap_ and _Infrastructure_ provider.
```

Comment:
On a getting started guide this needs spelling out, new folks will have no idea what this means.

Reply:
Really needs a diagram IMO.